tests/cpp/operator/test_cast_mxfp8.cu

```diff
@@ -429,18 +433,25 @@ void performTest_x2(const ProcessingMethod processing_method,
     }
 }
 
-std::vector<std::pair<size_t, size_t>> matrix_sizes = {
-    {1, 16},
-    {16, 48},
-    {65, 96},
-    {128, 128},
-    {256, 256},
-    {993, 512},
-    {768, 1024},
-    // {2048, 12288},
-    // {65536, 128},
-    // {16384, 1632},
-    // {16384, 6144},
+std::vector<std::vector<size_t>> matrix_sizes = {
+  {1, 16},
+  {32},
+  {65536},
+  {128, 128},
+  {256, 256},
+  {768, 1024},
+  {993, 512},
+  {256, 65536},
+  {2048, 12288},
+  {65536, 128},
+  {65536, 160},
+  {16384, 1632},
+  {1, 128},
+  {1, 1296},
+  {1, 32},
+  {5, 160},
+  {5, 4, 3, 160},
+  {217, 256},
 };
```

Comment:
I don't think we need all of these non-2D cases. The actually important kernel logic is in the 2D case, and all we're really testing in the non-2D cases is that they're correctly flattened into 2D:
```suggestion
std::vector<std::vector<size_t>> matrix_sizes = {
    {1, 16},
    {16, 48},
    {65, 96},
    {128, 128},
    {256, 256},
    {993, 512},
    {768, 1024},
    {256, 65536},
    {2048, 12288},
    {65536, 128},
    {65536, 160},
    {16384, 1632},
    {1024},
    {8, 32, 1024},
    {16, 8, 4, 512},
};
```